app/models/user.rb

```diff
@@ -81,13 +79,8 @@ def super_admin?
   end
 
   def need_two_factor_authentication?(request)
```

Comment:
This is flagged by the linter, however it relates to comment coming in 3... 2... 1...

Reply:
fixed that